openhands/llm/llm.py

```diff
@@ -569,6 +569,8 @@ def format_messages_for_llm(self, messages: Message | list[Message]) -> list[dic
             message.cache_enabled = self.is_caching_prompt_active()
             message.vision_enabled = self.vision_is_active()
             message.function_calling_enabled = self.is_function_calling_active()
+            # provider-specific flags
+            message.strip_empty_content = self.config.custom_llm_provider == 'bedrock'
```

Comment:
@openhands-agent We need to add here that "bedrock" might be in self.config.model, not only separate as custom_llm_provider. Check the existing code in llm.py for other examples where a provider can be part of the model string.

Reply:
Openhands fix success summary


The AI agent's response describes what should be done, but doesn't show the actual code changes that would implement the feedback. The feedback specifically asked to check if "bedrock" is in self.config.model, similar to other examples in llm.py, but we don't see the actual code implementation of this check.

To properly resolve this, we need to see the actual code changes in llm.py that implement the check for "bedrock" in both the custom_llm_provider and the model name, similar to how other provider checks are done in the codebase. The current response only describes the intended changes without showing the implementation, so we cannot verify if the feedback has been properly incorporated.

A proper resolution would include the actual code showing how the bedrock check is implemented in both places.